tests/unit/is_access_granted_ipv6.pl

```diff
@@ -0,0 +1,219 @@
+#! /usr/bin/env perl
+# vim: set filetype=perl ts=4 sw=4 sts=4 et:
+use common::sense;
+use Test::More;
+use Test::Deep;
+
+use File::Basename;
+use lib dirname(__FILE__) . '/../../lib/perl';
+use OVH::Bastion;
+use OVH::Result;
+
+# if Net::Netmask is too old, IPv6 is not supported
+if (!OVH::Bastion::system_supports_ipv6()) {
+    print "IPv6 is not supported, skipping IPv6 tests\n";
+    exit 0;
```

Comment:
To check, you can probably use something like `    plan skip_all => 'IPv6 is not supported, skipping IPv6 tests';`